code/modules/SCP/SCPs/SCP-049.dm

```diff
@@ -152,7 +152,7 @@ GLOBAL_LIST_EMPTY(scp049_1s)
 
 	if (!(target in orange(1, src)))
 		// moves slightly faster than humans
-		walk_to(src, target, 1, 0.9+config.run_speed)
+		walk_to(src, target, 1, config.run_speed)
```

Comment:
Alright, after serious consideration, I'm not going to merge this on SCP's being the same speed as humans. They might not have been in CB, but this is SS13. Balance is everything. I agree 0.9, almost an entire point, faster is too much, but we should see what it does on 0.2 first. If you could make SCP-106 that as well, please.

0.2 looks like a good starting point to nerf or buff further based on results, but to my understanding, 0.2 will give people a fighting chance to run and use airlocks to their advantage.